pkg/alert/ctl_alert_test.go

```diff
@@ -118,13 +118,13 @@ func TestAddSpecFlags(t *testing.T) {
 	expectedCmd.Flags().StringVar(&ctl.AlertImageVersion, "alert-image-version", ctl.AlertImageVersion, "Version of the Alert Image")
 	expectedCmd.Flags().StringVar(&ctl.CfsslImageName, "cfssl-image-name", ctl.CfsslImageName, "Name of Cfssl Image")
 	expectedCmd.Flags().StringVar(&ctl.CfsslImageVersion, "cfssl-image-version", ctl.CfsslImageVersion, "Version of Cffsl Image")
-	expectedCmd.Flags().StringVar(&ctl.BlackduckHost, "blackduck-host", ctl.BlackduckHost, "Host url of Blackduck")
-	expectedCmd.Flags().StringVar(&ctl.BlackduckUser, "blackduck-user", ctl.BlackduckUser, "Username for Blackduck")
-	expectedCmd.Flags().IntVar(&ctl.BlackduckPort, "blackduck-port", ctl.BlackduckPort, "Port for Blackduck")
 	expectedCmd.Flags().IntVar(&ctl.Port, "port", ctl.Port, "Port for Alert")
 	expectedCmd.Flags().BoolVar(&ctl.StandAlone, "stand-alone", ctl.StandAlone, "Enable Stand Alone mode")
+	expectedCmd.Flags().StringVar(&ctl.PVCStorageClass, "storage-class", ctl.PVCStorageClass, "StorageClass for the PVC")
 	expectedCmd.Flags().StringVar(&ctl.AlertMemory, "alert-memory", ctl.AlertMemory, "Memory allocation for the Alert")
 	expectedCmd.Flags().StringVar(&ctl.CfsslMemory, "cfssl-memory", ctl.CfsslMemory, "Memory allocation for the Cfssl")
+	expectedCmd.Flags().StringVar(&ctl.CfsslMemory, "pvc-memory", ctl.CfsslMemory, "Memory allocation for the PVC")
+	expectedCmd.Flags().StringSliceVar(&ctl.Environs, "environs", ctl.Environs, "Environment variables for the Alert")
```

Comment:
Are customers expected to set PUBLIC_HUB_WEBSERVER_HOST / ALERT_ENCRYPTION_PASSWORD / ALERT_ENCRYPTION_GLOBAL_SALT using --environs when deploying Alert with synopsysctl?

Reply:
I just talked to Paulo. The preferred way is for the user to enter them with --environs. If the user does not, then they will be prompted to enter them when they start using the Alert.

Reply:
Wouldn't it be better to add specific fields to the custom resource for this configuration and have specific switches in synopsysctl? I'm just thinking that it would be more user friendly. I doubt the customer is going to read the documentation and use --environs to set those.

Reply:
+1 to separate flags.  The would be translated to environs internally, but anywhere we can avoid arbitrary string formats is good.  Plus as separate flags we can make the fields required if needed

Reply:
Paulo said that even though they are required fields, he would prefer them to be entered as environs so that it's consistent with other environ variables that the customer can set.

Reply:
It's their decision I guess.  We want to get this into their hands and have them use it so they can decide things that provide the best customer experience for them.

Reply:
Senthil and I talked to Paulo about these environs being sensitive and how they should be put in a Secret. He now agrees that we should put them in spec fields. So I made the switch and created a Secret with them. I also made the Deployment add the environs from the secret.